.storybook/__snapshots__/Storyshots.test.js.snap

```diff
@@ -982,6 +982,179 @@ exports[`Storyshots Paragon Welcome 1`] = `
 </div>
 `;
 
+exports[`Storyshots StatusAlert Non-dismissible alert 1`] = `
+<div
+  className="alert fade alert-danger show"
+  role="alert"
+>
+  <div
+    className="alert-dialog"
+  >
+    You can't get rid of me!
+  </div>
+</div>
+`;
+
+exports[`Storyshots StatusAlert alert invoked via a button 1`] = `
+<div>
+  <div
```

Comment:
@arizzitano while playing with the screenreader yesterday, I think that I need to add an aria-hidden on the status alert that is tied with the "open" state field, but when I did my first attempt it didn't appear to work as I had expected.  Advice on the logic that should go here?

Reply:
Spoke with Mark -- he noted that we could leverage `display: 'none'` and `display: 'block'` for this instead of `aria-hidden` so I am looking into how to correctly integrate that logic into classNames